### PR TITLE
 Removing legacy commons-lang dependency

### PR DESCRIPTION
**Pull request by abhioncbr**
**Labels**
- `dependencies`
- `cleanup`

**Description**
- Removing legacy commons-lang dependency

### CODE REVIEW

**Review by Jackie-Jiang — APPROVED** [review]
Thanks for cleaning this up!

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/CpcSketchTest.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ public File createAvroFile()
       for (int i = 0; i < getCountStarResult(); i++) {
         // create avro record
         GenericData.Record record = new GenericData.Record(avroSchema);
-        record.put(ID, RandomUtils.nextInt(10));
+        record.put(ID, RandomUtils.nextInt(0, 10));
```
Comment (Jackie-Jiang) [inline pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/CpcSketchTest.java:184]: Seems `RandomUtils` is already deprecated. Can you replace it with a regular `Random`?

Comment (abhioncbr) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/CpcSketchTest.java:184]: I did notice the deprecation warning. I plan to use [commons-rng](https://commons.apache.org/proper/commons-rng/) as it's much more performant than `java.util.Random` in the follow-up PR. Thoughts?

Comment (Jackie-Jiang) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/CpcSketchTest.java:184]: Sounds good!

**Pull request merged**